features/bevragen/persoon/adressering/adres-regels/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,253 @@
+# language: nl
+
+Functionaliteit: autorisatie adressering adresregels Persoon
+
+    Achtergrond:
+      Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfplaats' gegevens
+      | gemeente van inschrijving (09.10) |
+      | 0599                              |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                              | waarde          |
+      | gemeentecode (92.10)              | 0599            |
+      | straatnaam (11.10)                | Borgesiusstraat |
+      | huisnummer (11.20)                | 103             |
+      | postcode (11.60)                  | 2497BV          |
+      | woonplaats (11.70)                | Scheveningen    |
+
+
+  Rule: Vragen met fields om een adresregel, wanneer de gebruiker niet geautoriseerd is voor alle velden waaruit aanhef wordt afgeleid, geeft een foutmelding
```

Comment:
hier wordt aanhef genoemd, maar in de scenario wordt adresregel1 gevraagd

Reply:
gecorrigeerd